.gitignore

```diff
@@ -1,2 +1,3 @@
 
 *.code-workspace
+localisation/DE+.ods
```

Comment:
Weird flex, but ok

Reply:
oh, that actually shouldn't have existed any more... --> deleted